Userland/Libraries/LibDeviceTree/DeviceTree.h

```diff
@@ -91,6 +91,13 @@ struct DeviceTreeProperty {
 };
 
 class DeviceTreeNodeView {
+    // FIXME: This struct only holds true for memory mapped nodes,
+    //        eg PCI devices have 96-bit addresses in their "reg" property
+    struct Reg {
+        FlatPtr address;
+        size_t length;
```

Comment:
This field should probably be called `size` instead, as that is what the spec calls these fields.